src/mokka/src/main/java/pl/hycom/mokka/stubbing/WireMockMappingSource.java

```diff
@@ -0,0 +1,68 @@
+package pl.hycom.mokka.stubbing;
+
+import com.github.tomakehurst.wiremock.standalone.MappingsSource;
+import com.github.tomakehurst.wiremock.stubbing.StubMapping;
+import com.github.tomakehurst.wiremock.stubbing.StubMappings;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.convert.ConversionService;
+import org.springframework.stereotype.Component;
+import pl.hycom.mokka.emulator.mock.MockConfigurationRepository;
+import pl.hycom.mokka.emulator.mock.model.MockConfiguration;
+
+import java.util.List;
+
+/**
+ * Implementements WireMock's {@link MappingsSource}
+ *
+ * @author adam.misterski@hycom.pl
+ */
+@Slf4j
+@Component
+public class WireMockMappingSource implements MappingsSource {
+
+    @Autowired
+    ConversionService conversionService;
```

Comment:
I think we can fully move to constructor injection now so could you please make it private final, remove existing constructor and use Lombok's RequiredArgsConstructor?